src/Spec2-Core/SpWindowPresenter.class.st

```diff
@@ -194,13 +194,13 @@ SpWindowPresenter >> centeredRelativeToTopWindow [
 	self centeredRelativeTo: self application topWindow
 ]
 
-{ #category : #api }
+{ #category : #updating }
```

Comment:
why `#close` is on updating?

Reply:
looks like it is unattended. I do not remember updating the protocol. We can set it back to #api